--- arch/arm/mach-rockchip/rk3588/Kconfig	2024-12-07 14:19:02.999475428 +0400
+++ arch/arm/mach-rockchip/rk3588/Kconfig	2024-10-23 11:41:36.821164715 +0400
@@ -5,6 +5,12 @@
 	select BOARD_LATE_INIT
 	help
 	  RK3588 EVB is a evaluation board for Rockchp RK3588.
+	  
+config TARGET_DCA_RK3588
+	bool "Rockchip DCA588 v2"
+	select BOARD_LATE_INIT
+	help
+	  RK3588 DCA is a multimedia board for Rockchp RK3588.
 
@@ -385,13 +367,11 @@
 	default 0x00a00000
 
 source "board/armsom/sige7-rk3588/Kconfig"
 source "board/edgeble/neural-compute-module-6/Kconfig"
 source "board/friendlyelec/cm3588-nas-rk3588/Kconfig"
 source "board/friendlyelec/nanopc-t6-rk3588/Kconfig"
 source "board/friendlyelec/nanopi-r6c-rk3588s/Kconfig"
 source "board/friendlyelec/nanopi-r6s-rk3588s/Kconfig"
 source "board/indiedroid/nova/Kconfig"
 source "board/pine64/quartzpro64-rk3588/Kconfig"
 source "board/turing/turing-rk1-rk3588/Kconfig"
@@ -399,6 +379,7 @@
 source "board/radxa/rock5b-rk3588/Kconfig"
 source "board/radxa/rock-5-itx-rk3588/Kconfig"
 source "board/rockchip/evb_rk3588/Kconfig"
+source "board/arm-sbc/dca_3588/Kconfig"
 source "board/rockchip/toybrick_rk3588/Kconfig"
 source "board/theobroma-systems/jaguar_rk3588/Kconfig"
 source "board/theobroma-systems/tiger_rk3588/Kconfig"
 
--- arch/arm/mach-rockchip/rk3568/Kconfig	2024-12-07 14:19:02.999475428 +0400
+++ arch/arm/mach-rockchip/rk3568/Kconfig	2024-11-08 17:42:33.520396479 +0400
@@ -22,29 +22,26 @@
 	help
 	  Hardkernel ODROID-M1 single board computer with a RK3568B2 SoC.

+config TARGET_DCA_RK3568
+	bool "ARM-SBC DCA-3568"
 	help
+	  ARM-SBC DCA-568 single board computer with a RK3568B2 SoC.
 
+config TARGET_DCA_RK3566
+	bool "ARM-SBC DCA-3566"
+	help
+	  ARM-SBC DCA-568 single board computer with a RK3568B2 SoC.
+	  
+config TARGET_NANO_RK3568
+	bool "ARM-SBC NANO-3568"
+	help
+	  ARM-SBC NANO 3568 single board computer with a RK3568 SoC.
+	  

@@ -80,11 +77,12 @@
 source "board/rockchip/evb_rk3568/Kconfig"
 source "board/anbernic/rgxx3_rk3566/Kconfig"
 source "board/hardkernel/odroid_m1/Kconfig"
 source "board/pine64/quartz64_rk3566/Kconfig"
 source "board/powkiddy/x55/Kconfig"
 source "board/radxa/zero3-rk3566/Kconfig"
 source "board/xunlong/orangepi-3b-rk3566/Kconfig"
+source "board/arm-sbc/dca_3568/Kconfig"
+source "board/arm-sbc/dca_3566/Kconfig"
+source "board/arm-sbc/nano_3568/Kconfig"
 
 endif
